core/account/indexer.go

```diff
@@ -216,14 +224,17 @@ func (m *Manager) upsertConfirmedAccountOutputs(ctx context.Context, outs []*acc
 		accountID = append(accountID, out.AccountID)
 		cpIndex = append(cpIndex, int64(out.keyIndex))
 		program = append(program, out.ControlProgram)
+		sourceID = append(sourceID, out.sourceID[:])
+		sourcePos = append(sourcePos, int64(out.sourcePos))
+		refData = append(refData, out.refData[:])
 	}
 
 	const q = `
 		INSERT INTO account_utxos (tx_hash, index, output_id, asset_id, amount, account_id, control_program_index,
-			control_program, confirmed_in)
+			control_program, confirmed_in, source_id, source_pos, ref_data_hash)
 		SELECT unnest($1::bytea[]), unnest($2::bigint[]), unnest($3::bytea[]), unnest($4::bytea[]),  unnest($5::bigint[]),
-			   unnest($6::text[]), unnest($7::bigint[]), unnest($8::bytea[]), $9
-		ON CONFLICT (tx_hash, index) DO NOTHING
```

Comment:
Why did we lose the `ON CONFLICT` clause? It's still present in https://github.com/chain/chain/pull/665